src/store/validation/index.ts

```diff
@@ -106,31 +106,39 @@ export const actions: ActionTree<IValidationState, IRootState> = {
   },
 
   /**
-   * Validate blocks from backend SAVE action, pick info from from active flow's vendor_metadata:
+   * Validate blocks from backend SAVE action.
+   */
+  async validate_allBlocksFromBackend({dispatch}): Promise<void> {
+    await dispatch('validate_fromBackend', {type: 'block'})
+    await dispatch('validate_fromBackend', {type: 'resource'})
+  },
+
+  /**
+   * Validate block/resource form backend, pick info from active flow's vendor_metadata:
    * floip: {
```

Comment:
- add `ui_metadata`
- optional: instead of nesting use chaining like `vendor_metadata.floip.ui_metadata.validation_results: {...}`